policy/release/sbom_cyclonedx.rego

```diff
@@ -68,3 +68,130 @@ deny contains result if {
 	count(object.get(s, "components", [])) == 0
 	result := lib.result_helper(rego.metadata.chain(), [])
 }
+
+# METADATA
+# title: Allowed
+# description: >-
+#   Confirm the CycloneDX SBOM contains only allowed packages. By default all packages are allowed.
+#   Use the "disallowed_packages" rule data key to provide a list of disallowed packages.
+# custom:
+#   short_name: allowed
+#   failure_msg: "Package is not allowed: %s"
+#   solution: >-
+#     Update the image to not use a disallowed package.
+#   collections:
+#   - redhat
```

Comment:
what happens if there is no sbom and the `Redhat` collection` is used?

Reply:
The [sbom_cyclonedx.found](https://enterprisecontract.dev/docs/ec-policies/release_policy.html#sbom_cyclonedx__found) rule in the current redhat collection already fails if it is missing.